ecommerce/src/Relecloud.Web/Services/SqlDatabaseConcertRepository/ConcertDataContext.cs

```diff
@@ -32,9 +29,9 @@ public void Initialize()
                 var databaseName = this.Database.GetDbConnection().Database;
                 // Note: the cast to string is to work around an issue in Entity Framework Core 2.0 with string interpolation.
                 // See https://github.com/aspnet/EntityFrameworkCore/issues/9734.
-                this.Database.ExecuteSqlCommand((string)$"ALTER DATABASE [{databaseName}] SET CHANGE_TRACKING = ON (CHANGE_RETENTION = 2 DAYS, AUTO_CLEANUP = ON)");
+                this.Database.ExecuteSqlRaw((string)$"ALTER DATABASE [{databaseName}] SET CHANGE_TRACKING = ON (CHANGE_RETENTION = 2 DAYS, AUTO_CLEANUP = ON)");
```

Comment:
I don't know if the cast to string is still required here as a workaround for the (old) EF string interpolation issue mentioned above?